src/strategies/sushiswap/index.ts

```diff
@@ -2,21 +2,47 @@ import { getAddress } from '@ethersproject/address';
 import { formatUnits } from '@ethersproject/units';
 import { subgraphRequest } from '../../utils';
 
+// Minichef: https://github.com/sushiswap/sushiswap-interface/blob/master/src/services/graph/fetchers/masterchef.ts
+// Exchange: https://github.com/sushiswap/sushiswap-interface/blob/master/src/services/graph/fetchers/exchange.ts
+
+const theGraph_baseUrl = 'https://api.thegraph.com/subgraphs/name/'
 const SUSHISWAP_SUBGRAPH_URL = {
-  '1': {
-    exchange: 'https://api.thegraph.com/subgraphs/name/sushiswap/exchange',
-    masterChef: 'https://api.thegraph.com/subgraphs/name/sushiswap/master-chef'
+  exchange: {
+    '1': 'sushiswap/exchange',
+    '100': 'sushiswap/xdai-exchange',
+    '137': 'sushiswap/matic-exchange',
+    '250': 'sushiswap/fantom-exchange',
+    '42161': 'sushiswap/arbitrum-exchange',
+    '1285': 'sushiswap/moonriver-exchange',
+    '42220': 'jiro-ono/sushitestsubgraph',
+    '122': 'sushiswap/fuse-exchange',
+    '1666600000': 'sushiswap/harmony-exchange',
+    '56': 'sushiswap/bsc-exchange',
+    '43114': 'sushiswap/avalanche-exchange',
+    '66': 'okex-exchange/oec',
+    '128': 'heco-exchange/heco'
   },
-  '100': {
-    exchange: 'https://api.thegraph.com/subgraphs/name/sushiswap/xdai-exchange',
-    masterChef:
-      'https://api.thegraph.com/subgraphs/name/matthewlilley/xdai-minichef'
+  masterChef: {
+    '1': 'sushiswap/master-chef',
+    '100': 'matthewlilley/xdai-minichef', // Could be replaced by 'sushiswap/xdai-minichef'
+    '137': 'sushiswap/matic-minichef',
+    '250': 'sushiswap/fantom-minichef',
+    '42161': 'matthewlilley/arbitrum-minichef',
+    '1285': 'sushiswap/moonriver-minichef',
+    '42220': 'sushiswap/celo-minichef-v2',
+    '122': 'sushiswap/fuse-minichef',
+    '1666600000': 'sushiswap/harmony-minichef'
+  }, 
+  masterChefV2: {
+    '1': 'sushiswap/master-chefv2'
   }
-};
+}
+
+
 const PAGE_SIZE = 1000;
 
 export const author = 'vfatouros';
```

Comment:
```suggestion
export const author = 'maxaleks';
```

Reply:
hehe you could have put jo-chemla. 
Thanks for the merge!